index.js

```diff
@@ -8,6 +8,15 @@ const verifySignatures = (msgs) => {
   return v.verifySignatures(jsonMsgs);
 };
 
+const validateSingle = (msg, previous) => {
+  const jsonMsg = JSON.stringify(msg, null, 2);
+  if (previous !== undefined) {
```

Comment:
Oh that's great to know, thanks! I'll make the same change for `validateBatch` as well.

Reply:
Hmm but wait, this isn't quite right. I think we actually want `if (previous)` instead of `if (!previous)`. The second option will _only_ match on `null`, `undefined` etc. (the opposite of what we want). I realised this when I made the change and ran into errors in the tests.